pkg/conf/schedulerconf.go

```diff
@@ -73,19 +74,24 @@ const (
 	CMKubeQPS   = PrefixKubernetes + "qps"
 	CMKubeBurst = PrefixKubernetes + "burst"
 
+	// admissioncontroller
+	PrefixAMFiltering               = PrefixAdmissionController + "filtering."
+	AMFilteringGenerateUniqueAppIds = PrefixAMFiltering + "generateUniqueAppId"
```

Comment:
NOTE: This is needed as the specific algorithm used for generating the appID is controlled by an admission-controller specific property. We use that here as well to maintain backwards compatibility. The value is duplicated form am_conf.go as we would create a circular package reference otherwise.